thorlcr/graph/thgraphslave.cpp

```diff
@@ -202,7 +202,7 @@ void CSlaveActivity::setLookAhead(unsigned index, IStartableEngineRowStream *loo
 IStrandJunction *CSlaveActivity::getOutputStreams(CActivityBase &ctx, unsigned idx, PointerArrayOf<IEngineRowStream> &streams, const CThorStrandOptions * consumerOptions, bool consumerOrdered, IOrderedCallbackCollection * orderedCallbacks)
 {
     // Default non-stranded implementation, expects activity to have 1 output.
-    assertex(!idx);
+//    assertex(!idx);
```

Comment:
I should remove.
idx is the inputs output index, which be != 0  for an activity with just 1 input. e.g. any act fed by >0 output of a splitter.